types/parse/index.d.ts

```diff
@@ -830,11 +836,31 @@ subscription.on('close', () => {});
         function beforeFind(arg1: any, func?: (request: BeforeFindRequest) => Promise<Query> | Query): void;
         function afterFind(arg1: any, func?: (request: AfterFindRequest) => Promise<any> | any): void;
         function define(name: string, func?: (request: FunctionRequest) => Promise<any> | any): void;
+        /**
+         * Gets data for the current set of cloud jobs.
+         * @returns A promise that will be resolved with the result of the function.
+         */
+        function getJobsData(): Promise<any>;
+        /**
+         * Gets job status by Id
+         * @param jobStatusId The Id of Job Status.
+         * @returns Status of Job.
+         */
+        function getJobStatus(jobStatusId: string): Promise<any>;
```

Comment:
```suggestion
        function getJobStatus(jobStatusId: string): Promise<Object>;
```

Reply:
Do you mean `object` (with lowercase) or `Parse.Object`? 
If the first one, I think it's not optimal as it will limit accessing properties in the returned object, that's why the rest of the Promises are returning `any`.
If the latter, it's not in the [docs](https://parseplatform.org/Parse-SDK-JS/api/2.7.0/Parse.Cloud.html#.getJobsData), so if you know this I can change it.

Reply:
I meant `Parse.Object`. In fact, that docs seems to be different from [source code (where I found that type)](https://github.com/parse-community/Parse-SDK-JS/blob/05e8c79e17c8d7f30280303c41848dcd15b6d5cb/src/Cloud.js#L108).

Reply:
Got it. Thanks.